apps/test/unit/schoolInfo/hooks/useSchoolInfoTest.jsx

```diff
@@ -311,6 +311,26 @@ describe('useSchoolInfo', () => {
         expect(schoolNameSessionStorageCalls[1][1]).toBe('Super Cool School');
       });
     });
+
+    it('should reset school info to initial state if passed', async () => {
+      await act(async () => {
+        hook.current.setSchoolZip('90210');
+      });
+      expect(hook.current.schoolZip).toBe('90210');
+      act(async () => {
```

Comment:
this `act` and the one on line 326 are async and need `await`. that will fix the test order issue

Reply:
Looks like this ended up being a combination of my test missing `await`s and the analytics ones missing `await`s? That tracks as I was definitely modeling parts of this test off the analytics tests 😅 

Thanks for fixing!